### DISCUSSION TITLE
Inviting new user using POST `/users/invite` endpoint

### DISCUSSION THREAD

**Discussion by brymut** [directus/directus#5778] [Archive]
Hi, trying to set up inviting new users. However when I send a `POST` to `/users/invite` with `email` , `role` and `invite_url` I receive a 400 bad request with the error message:
``` [{message: ""invite_url" is not allowed", extensions: {code: "INVALID_PAYLOAD"}}] ```

Currently, I'm setting `invite_url` as `localhost:3000/invite` which contains the rest of the invite flow after clicking through the email I have also set the `ENV USER_INVITE_URL_ALLOW_LIST` to contain localhost:3000/invite that is `"localhost:3000/invite"` but still receiving the same error. So I also tried other URLs to see if it was just localhost but I still get the same issue with other fqdn.

eg: I have set `ENV USER_INVITE_URL_ALLOW_LIST=["https://exampleone.com","https://anotherone.com"]` in my docker file and the server starts up successfully and then I send the following POST request and still get a 400 response with the same error message:

![Screenshot 2021-05-20 at 06 47 37](https://user-images.githubusercontent.com/14836056/118916329-64c67680-b937-11eb-82ce-f089b15bee5d.png)

May I please get some help?

**Comment by rijkvanzanten** [comment]
Are you on the latest version of Directus?

**Reply by brymut** [reply]
I think so, I'm using docker tagged `directus/directus:v9.0.0-rc.24`

**Reply by rijkvanzanten** [reply]
Then the answer is no 🙂 We're currently on rc.69: https://github.com/directus/directus/releases

**Reply by brymut** [reply]
Yikes, I don't know how I missed that. Thanks for your patience, I'll update and let you know.
